src/ui/components/DashKit/plugins/Title/DashAnchorLink/DashAnchorLink.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import type {PluginTitleProps} from '@gravity-ui/dashkit';
+import {Link} from '@gravity-ui/uikit';
+import block from 'bem-cn-lite';
+import {useHistory, useLocation} from 'react-router';
+
+const b = block('dashkit-plugin-container');
+
+interface DashAnchorLinkProps {
+    size: PluginTitleProps['data']['size'];
+    x: number;
+    to: string;
+}
+
+const DashAnchorLink = ({size, x, to}: DashAnchorLinkProps) => {
+    const location = useLocation();
+    const history = useHistory();
+    const hash = `#${encodeURIComponent(to)}`;
+    const isLinkVisible = false || location.hash === hash;
+
+    return (
+        <Link
+            onClick={(e) => {
+                e.preventDefault();
```

Comment:
If this is a link, then it should open in a new tab by cmd + click (the table of contents does so)